docs/user-guide/api-mediation/api-mediation-multi-tenancy.md

```diff
@@ -117,12 +114,9 @@ Use the following example as a template for how to set the value of this propert
 
 **Example:**
 ```
-components.cloud-gateway.apiml.service.additionalRegistration:
-    # central API ML (in HA, for non-HA mode use only 1 hostname)
-       - discoveryServiceUrls:      https://sys1:{discoveryServicePort}/eureka/,https://sys2:{discoveryServicePort}/eureka/
- 	    routes:
-              - gatewayUrl: /
-                serviceUrl: /
+components.gateway.apiml.service.additionalRegistration:
+       # central API ML (in HA, for non-HA mode use only 1 hostname)
+       - discoveryServiceUrls: https://sys1:{discoveryServicePort}/eureka/,https://sys2:{discoveryServicePort}/eureka/
```

Comment:
Please add the note that `sys1` and `sys2` are referring to a different LPAR inside one sysplex. It was confusing in the past  (see https://github.com/zowe/docs-site/pull/3951/files#diff-6a06e4905263943fe346deef59375628c08549468c145a3ebd34cdc088182e3b). It issue is that this looks like we are talking about 2 APIML instances, but the example shows just all instances  of DS in one particular APIML instance.